tools/config_template.py

```diff
@@ -19,6 +19,10 @@
 # This engine must exist, or nothing can be compiled.
 NODE_JS = '{{{ NODE }}}' # executable
 
+# Location of the the TypeScript compiler used for generating optional
+# TypeScript definitions.
+TSC = '{{{ TSC }}}' # executable
```

Comment:
I wouldn't worry about adding that here in this template file.

This setting is optional anyway right?   Lets just make leave it out for simplicity for now.